torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2860,6 +2860,12 @@ def sample_inputs_bernoulli(self, device, dtype, requires_grad, **kwargs):
         samples.append(SampleInput(t))
     return tuple(samples)
 
+def error_inputs_bernoulli(op_info, device, **kwargs):
+    x = torch.rand((1,), device=device).expand((6,))
```

Comment:
Add a comment describing what about this input is unsupported

Reply:
Added!